ui/pages/index.tsx

```diff
@@ -290,12 +302,29 @@ function PublicLandingPage() {
                 Piloter l’apprentissage
               </Heading>
               <Text color="#3A3A3A" fontSize="zeta">
-                Grâce aux données en temps réel dans un territoire ou un secteur économique
+                Pour adapter les besoins et l’offre de formation sur un territoire ou sur un secteur économique.
               </Text>
             </VStack>
           </Stack>
         </Container>
       </Box>
+
+      <Container maxW="xl" bg="#FEF9F7" px="14" py="10" my="12">
+        <Text fontWeight="bold" color="blue_cumulus_main" fontSize="gamma">
+          Suivez nos actualités sur LinkedIn “Mission Interministérielle pour l’apprentissage”
```

Comment:
Les guillemets français sont plutôt comme ça mais c'est ignorable.

```suggestion
          Suivez nos actualités sur LinkedIn «&nbsp;Mission Interministérielle pour l’apprentissage&nbsp;»
```